### DISCUSSION TITLE
Setup HTTPS

### DISCUSSION THREAD

**Discussion by virus4sepp** [garethgeorge/backrest#404] [Q&A] [answered]
Hello,

just a short Question, is it possible to active HTTPS for backrest? If yes how can i do it or is there any other way?

Thanks and Greetings Christoph

**Comment by garethgeorge** [comment, marked as answer]
Hey, for local access backrest is http only. If you want to expose backrest over the internet I recommend using a reverse proxy e.g. caddy.

**Reply by virus4sepp** [reply]
Thanks for you fast answer.
